.github/workflows/create-release-package.yml

```diff
@@ -78,7 +78,16 @@ jobs:
       env:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
       with:
-        upload_url: ${{ steps.create_release.outputs.upload_url }} # This pulls from the CREATE RELEASE step above, referencing it's ID to get its outputs object, which include a `upload_url`. See this blog post for more info: https://jasonet.co/posts/new-features-of-github-actions/#passing-data-to-future-steps 
+        upload_url: ${{ steps.create_release.outputs.upload_url }} # This pulls from the CREATE RELEASE step above, referencing it's ID to get its outputs object, which include a `upload_url`. See this blog post for more info: https://jasonet.co/posts/new-features-of-github-actions/#passing-data-to-future-steps
         asset_path: ./atlas-osb.${{ steps.semver.outputs.version }}.tar.gz
         asset_name: atlas-osb.${{ steps.semver.outputs.version }}.tar.gz
         asset_content_type: application/tgz
+        
+    - name: Push Atlas-osb to Registry
+      uses: docker/build-push-action@v1
+      with:
+        username: ${{ secrets.REGISTRY_USERNAME }}
+        password: ${{ secrets.REGISTRY_PASSWORD }}
+        registry: ${{ secrets.REGISTRY }}
+        repository: ${{ secrets.REGISTRY_REPO }}
+        tags: ${{ steps.semver.outputs.version }}, latest
```

Comment:
[q] where is this variable defined?

Reply:
from action output https://github.com/mongodb/atlas-osb/blob/60647aca02adb8034e2fb65950d5bb4142465603/.github/workflows/create-release-package.yml#L45
https://github.com/leo-ri/increment-semver/blob/b43acb48b88f31fe1d5f7579baeada2ec9cdc58b/entrypoint.sh#L87